language/de-DE/com_content.ini

```
@@ -93,4 +94,5 @@ COM_CONTENT_TRANSITION="Status"
 COM_CONTENT_VOTES="Abstimmung"
 COM_CONTENT_VOTES_COUNT="Abstimmungen: %s"
 COM_CONTENT_WORKFLOW="Workflow"
+COM_CONTENT_WORKFLOW_STAGE="Workflow Stage"
```

```suggestion
COM_CONTENT_WORKFLOW_STAGE="Workflow-Phase"
```
https://github.com/joomlagerman/joomla/blob/1f5d03b967c443a1e291a4bbf79205f106fa094a/administrator/language/de-DE/com_content.ini#L195